NEWS.md

```diff
@@ -1,11 +1,201 @@
 # NEWS
 
-## Unreleased
+## 0.27.0
+
+### Features
+
+- Pipeline store writes (#3084) (#3177)
+- GraphQL route to expose call cache contents by block (#3343)
+- Ethereum Transaction Receipts in Event Handlers (#3373) (#3444)
+- Bump max `apiVersion` to 0.0.7 and `specVersion` to 0.0.5 (#3421)
+- Add Firehose blockstream metrics (#3435)
+- Add `_change_block` to entities GraphQL `input` type filter (#3186)
+- Add publicProofOfIndexing resolver to the indexing status API (#3358)
+- Handle Ethereum Firehose decoding issues without crashing (#3483)
+- Make Ethereum call revert detection more sensitive (#3490)
+- New stable-hash implementation (#3485)
+- Support TLS PostgreSQL connections in notification_listener (#3503)
+- Arweave integration (#3465)
+- Add ipfs.getBlock (#3577)
```

Comment:
Lets not advertise this, it's mostly a hack to be eventually superseded by file data sources